packages/plugin-ngrok/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@shopify/plugin-ngrok",
+  "version": "0.2.9",
```

Comment:
Have we made a decision around aligning the versioning the official plugins with the other packages?

Reply:
The idea is that official plugins will be aligned too yes. I added the plugin to the changeset list so in the next release after merge it should be updated to 3.x.x